tests/console/yast2_rmt.pm

```diff
@@ -0,0 +1,85 @@
+# SUSE's openQA tests
+#
+# Copyright © 2009-2013 Bernhard M. Wiedemann
+# Copyright © 2012-2019 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary : Test for the yast2-rmt module
+# Maintainer Jonathan Rivrain <JRivrain@suse.com>
+
+use base "console_yasttest";
+use strict;
+use utils;
+use testapi;
+
+
+sub password_twice {
+    type_string "$password";
+    send_key "tab";
+    type_string "$password";
+    send_key "alt-o";
+}
+
+sub test_ui {
+    script_run("yast2 rmt; echo yast2-rmt-server-status-\$? > /dev/$serialdev", 0);
+    assert_screen "yast2_rmt_registration";
+    send_key "alt-n";
+    assert_screen "yast2_rmt_ignore_registration_dialog";
+    send_key "alt-i";
+    send_key "ret";
+    assert_screen "yast2_rmt_db_password";
+    send_key "alt-p";
+    type_string "$password";
+    send_key "alt-n";
+    assert_screen "yast2_rmt_db_root_password";
+    password_twice;
+    assert_screen "config_written_successfully";
+    send_key "alt-o";
+    assert_screen "yast2_rmt_ssl";
+    send_key "alt-o";
+    send_key_until_needlematch('yast2_rmt_common_name_empty', 'backspace');
+    type_string "rmt1.susetest.org";
+    send_key "alt-d";
+    assert_screen "yast2_rmt_ssl_add_name";
+    type_string "localhost";
+    send_key "alt-o";
+    send_key "alt-n";
+    assert_screen "yast2_rmt_ssl_CA_password";
+    password_twice;
+    assert_screen "yast2_rmt_firewall";
+    send_key "spc";
+    send_key "alt-n";
+    assert_screen "yast2_rmt_service_status";
+    send_key "alt-n";
+    assert_screen "yast2_rmt_config_summary";
+    send_key "alt-f";
+    wait_serial('yast2-rmt-server-status-0', 60) || die "'yast2 rmt' didn't finish";
+}
+
+sub test_config {
+    my @unit = ("rmt-server.service", "rmt-server-sync.timer", "rmt-server-mirror.timer");
+    for (@unit) {
+        script_run("systemctl is-active $_") && die "The systemd unit $_ is not active";
+    }
+    script_run("firewall-cmd --list-services |grep ' http' |grep ' https'")             && die "The firewall ports are not opened";
```

Comment:
you could use extension grep: `grep -E 'word1.*word2|word2.*word1' logs` and not even care about the second part of the or because they are ordered, so just `grep -E 'http.*https'`

Reply:
For the case we were discussing in person, for example, `tigervnc-https`, we can do `grep -E 'http[[:space:]]https'`

Reply:
I applied your suggestion.